tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 85.44, "AMD": 84.77, "ARM": 83.42}
+COVERAGE_DICT = {"Intel": 85.44, "AMD": 84.68, "ARM": 83.52}
```

Comment:
pls coverage commit to integ tests commit (or some other)

Reply:
Done.